Documentation/ceph-osd-mgmt.md

```diff
@@ -119,13 +119,15 @@ If you want to remove OSDs by hand, continue with the following sections. Howeve
 
 If the OSD purge job fails or you need fine-grained control of the removal, here are the individual commands that can be run from the toolbox.
```

Comment:
Above (around line 112) how about adding a comment about how to preserve the PVC with the new argument? Also maybe set the param to false in osd-purge.yaml so it's obvious where it can be changed to true.

Reply:
Indeed, I updated `osd-purge.yaml` to mention `--preserve-pvc` option.